syntax = "proto3";

package proto;

service Searcher {
    rpc Search(SearchRequest) returns (SearchReply) {}
}

message SearchRequest{
    string city = 1;
}

message SearchReply{
    bool correct = 1;
}